gae-interop-testing/gae-jdk8/build.gradle

```diff
@@ -43,17 +43,15 @@ repositories {
 apply plugin: 'com.google.cloud.tools.appengine'  // App Engine tasks
 
 dependencies {
-    providedCompile group: 'javax.servlet', name: 'servlet-api', version:'2.5'
-    compile 'com.google.appengine:appengine-api-1.0-sdk:1.9.59'
-    // Deps needed by all gRPC apps in GAE
-    compile libraries.google_api_protos
-    compile project(":grpc-okhttp")
-    compile project(":grpc-protobuf")
-    compile project(":grpc-stub")
-    compile (project(":grpc-interop-testing")) {
-        exclude group: "io.grpc", module: "grpc-netty-shaded"
+    implementation project(':grpc-netty'),
+            project(":grpc-stub"),
+            libraries.protobuf,
+            libraries.truth
+    implementation (project(':grpc-interop-testing')) {
+        exclude group: 'io.grpc', module: 'grpc-alts'
     }
-    compile libraries.netty_tcnative
```

Comment:
This looks like it was removed. It seems it should be a runtimeOnly dep.

Reply:
Yeah, we use tcnative on GAE Java 8 and want to make sure it works, since most people will be using grpc-netty-shaded which includes it. Since v1.25.0 the test probably passes without tcnative, though, since GAE Java 8 includes Conscrypt. But we still want to test with tcnative.

Reply:
I see. Add it add as runtimeOnly dep in #7001 now.